composer.json

```diff
@@ -3,13 +3,18 @@
 	"type": "typo3-cms-extension",
 	"description": "Includes a current version of ADOdb, a database abstraction library for PHP, for further use with EXT:dbal",
 	"homepage": "https://typo3.org",
-	"license": ["GPL-2.0+"],
+        "license": ["GPL-2.0-or-later"],
         "keywords" : [ "database", "abstraction", "layer", "library", "typo3" ],
 	"require": {
 		"typo3/cms-core": "~8.4"
 	},
 	"replace": {
 		"adodb": "*",
```

Comment:
I‘d remove this replacement as well

Reply:
ha, thanks. I actually wanted to remove THIS exact section